BenchmarkTest00228.py

```
@@ -0,0 +1,15 @@
+#{fact rule=autoescape-disabled@v1.0 defects=1}
+
+# cf. https://github.com/PyCQA/bandit/blob/02bad2e42311f420aef52dcd9806d66516ef594d/examples/jinja2_templating.py
+
+import jinja2
+from jinja2 import Environment, select_autoescape
+templateLoader = jinja2.FileSystemLoader( searchpath="/" )
+something = ''
+
+
+# ruleid:autoescape-disabled
+Environment(loader=templateLoader, load=templateLoader, autoescape=something)
```

Ruff (0.13.1)

Using jinja2 templates with `autoescape=False` is dangerous and can lead to XSS. Ensure `autoescape=True` or use the `select_autoescape` function.

(S701)